packages/network-of-terms-cli/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@netwerk-digitaal-erfgoed/network-of-terms-cli",
+  "private": true,
+  "type": "module",
+  "description": "",
```

Comment:
Perhaps: 'Query the Network of Terms via the command line' or 'Network of Terms command-line interface'?